salt/orch/update.sls

```diff
@@ -0,0 +1,121 @@
+# Generic Updates
+update_pillar:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_pillar
```

Comment:
and, it seems this was added in 2016.11.4.

https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.saltutil.html#salt.modules.saltutil.refresh_grains